storage/innobase/dict/dict0crea.cc

```diff
@@ -470,6 +478,12 @@ dict_build_tablespace_for_table(
 			DICT_TF2_FLAG_UNSET(table,
 					    DICT_TF2_FTS_AUX_HEX_NAME););
 
+	if (node && (node->mode == FIL_ENCRYPTION_ON ||
+			(node->mode == FIL_ENCRYPTION_DEFAULT &&
+			 (srv_encrypt_tables == SRV_ENCRYPT_TABLES_ONLINE_TO_KEYRING
```

Comment:
This is a general question. Since we have all kinds of encryptions, is there a I_S view that shows the status/type of encryption for each table.

1. master key encryption
2. keyring key encryption
3. in-progress (from mK to keyring key)
4 in-progress (from keyring key to mk)
5. unencrypted.